src/log.js

```diff
@@ -548,6 +548,19 @@ export function rethrowAsync(var_args) {
 }
 
 
+/**
+ * Rethrows the error immediately. Required because async errors thrown during
+ * tests leak to subsequent tests. DO NOT USE IN RUNTIME CODE.
+ * @visibleForTesting
+ * @param {...*} var_args
+ */
+export function rethrowAsyncForTests(var_args) {
```

Comment:
can we move this method to init_tests? or any where under test folders.

Reply:
I meant to do this once I got everything to work. Done.